sihl/src/sihl.mli

```diff
@@ -359,21 +599,11 @@ module Schedule : sig
 end
 
 module Random : sig
-  include module type of Core_random
-end
+  (** [base64 n] returns a Base64 encoded string containing [n] random bytes. *)
+  val base64 : int -> string
 
-module Contract : sig
-  module Cache = Contract_cache
-  module Database = Contract_database
-  module Email = Contract_email
-  module Email_template = Contract_email_template
-  module Http = Contract_http
-  module Migration = Contract_migration
-  module Password_reset = Contract_password_reset
-  module Queue = Contract_queue
-  module Random = Contract_random
-  module Schedule = Contract_schedule
-  module Storage = Contract_storage
-  module Token = Contract_token
-  module User = Contract_user
+  (** [bytes n] returns a byte sequence as string with [n] random bytes. In most
```

Comment:
It is returned as a `char list` though, not a string. Should we change the return type?

Reply:
Yes, I return a `string` now.

Reply:
Thx